### PR TITLE
Mobile ads fix

### PR DESCRIPTION
**Pull request by uplne**
Ads are not being served for mobile devices on fronts:
screen shot 2015-08-25 at 14 12 20

Bug has something to do with adding two divs for adslot for mobile and tablet+ to be able to deliver responsive experience.

This will fix the issue however it will turn off responsive behaviour till we will find what is causing the issue.

@Calanthe @jbreckmckye

### CODE REVIEW

File: static/src/javascripts/test/spec/common/commercial/slice-adverts.spec.js [hunk 1]
```diff
@@ -15,12 +15,21 @@ describe('Slice Adverts', function () {
         },
         $fixtureContainer,
         injector = new Injector(),
-        sliceAdverts, config;
+        sliceAdverts, config, detect;
 
     beforeEach(function (done) {
-        injector.test(['common/modules/commercial/slice-adverts', 'common/utils/config'], function () {
+        /*injector.mock({
```
Comment (jbreckmckye) [inline static/src/javascripts/test/spec/common/commercial/slice-adverts.spec.js]: Should this be deleted?

File: static/src/javascripts/test/spec/common/commercial/slice-adverts.spec.js [hunk 2]
```diff
@@ -28,6 +37,10 @@ describe('Slice Adverts', function () {
                 standardAdverts: true
             };
 
+            detect.getBreakpoint = function () {
```
Comment (jbreckmckye) [inline static/src/javascripts/test/spec/common/commercial/slice-adverts.spec.js]: Isn't this already set in the beforeEach?

File: static/src/javascripts/test/spec/common/commercial/slice-adverts.spec.js [hunk 3]
```diff
@@ -42,16 +55,19 @@ describe('Slice Adverts', function () {
...
 
         fastdom.defer(function () {
-            expect(qwery('.ad-slot', $fixtureContainer).length).toEqual(6);
+            expect(qwery('.ad-slot', $fixtureContainer).length).toEqual(3);
             done();
         });
     });
 
     it('should remove the "fc-slice__item--no-mpu" class', function (done) {
+        detect.getBreakpoint = function () {
```
Comment (jbreckmckye) [inline static/src/javascripts/test/spec/common/commercial/slice-adverts.spec.js]: Do we need to stub this again here when we already mock it with the injector?

**Comment by uplne** [comment]
Nice catch :-) I was rushing to deploy

**Comment by jbreckmckye** [comment]
+1, looks good

**Pull request merged**